lab_2_keywords_cooccurrence/main.py

```diff
@@ -29,7 +36,34 @@ def extract_candidate_keyword_phrases(phrases: Sequence[str], stop_words: Sequen
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not (phrases and stop_words):
+        return None
+    if not (isinstance(phrases, list) and isinstance(stop_words, list)):
+        return None
+    for phrase in phrases:
+        if not isinstance(phrase, str):
+            return None
+    phrase_list = []
+    list_for_tuples = []
+    candidate_keywords_phrases = []
+    for phrase in phrases:
+        phrase = phrase.lower()
+        phrase_list.append(phrase.split())
+    for phrase in phrase_list:
+        for word in phrase:
+            if word not in stop_words:
+                list_for_tuples.append(word)
+            else:
+                phrase_tuple = tuple(list_for_tuples)
+                if len(phrase_tuple) != 0:
```

Comment:
`if len(phrase_tuple)` would be enough to check

Reply:
done